dev/PushNotifications/PushNotificationTelemetry.h

```diff
@@ -116,21 +118,61 @@ class PushNotificationTelemetry : public wil::TraceLoggingProvider
         return m_eventCount;
     }
 
-    wchar_t m_appUserModelId[APPLICATION_USER_MODEL_ID_MAX_LENGTH] = {};
+    inline bool IsPackagedApp()
+    {
+        static bool isPackagedApp = AppModel::Identity::IsPackagedProcess();
```

Comment:
Is this a valid approach in COM /Winrt? 

It's valid and thread safe C++ code but I'm concerned that it may not play well with Com/WinRT.